Added withdraw-deposit

mileseu opened the pull request:
Hi all,
After ditching #7 , I've piggybacked Carolina's code (Thanks Carolina) written for transfer #8 and written methods for withdraw & deposit using the same structure and logic.
There's a fair bit of repeated code, and it's inelegant but it runs.
In the interest of getting my very much outstanding tickets from last sprint done, I thought it would be best to try and get this merged asap before this week's sprint.
Cheers,
Miles

mileseu1 committed:
Added withdraw-deposit

Added withdraw deposit methods using logic from Carolina's transfer methods.

carolinamartes commented on newbank/server/Customer.java:
```diff
@@ -24,6 +24,16 @@ public void printTransferableFromAccounts(double requestAmount) {
 		MenuPrinter.printTransferableFrom(transferableFromAccounts);
 	}
 
+	public void printDepositToAccounts(double requestAmount) {
+		ArrayList<Account> depositToAccounts = new ArrayList<Account>();
+		for(Account a : accounts) {
+			if (a.getCurrentBalance() >= requestAmount){
+				depositToAccounts.add(a);
+			}
+		}
+		MenuPrinter.printDepositTo(depositToAccounts);
+	}
```
not sure if I'm reading this wrong but don't we want to check `getCurrentBalance() >= requestAmount` only on withdraw, not deposit?

mileseu replied:
Whoops, that's what I get for copying and pasting without looking properly. Removing requestAmount :)

carolinamartes replied:
nice, in that case we can also remove `ArrayList<Account> depositToAccounts = new ArrayList<Account>();` and just call `MenuPrinter.printDepositTo(accounts);` Otherwise currently empty I believe

mileseu replied:
Thank you, done and tested :)

mileseu1 committed:
Removed printDepositToAccounts requestAmount

carolinamartes commented:
would be great to do some refactoring to streamline the repetitive bits but looks good to me 👍

mileseu commented:
![](https://github.trello.services/images/mini-trello-icon.png) [- Users can withdraw money from an account](https://trello.com/c/AVGj3Ufh/29-users-can-withdraw-money-from-an-account)

mileseu commented:
![](https://github.trello.services/images/mini-trello-icon.png) [- Users can deposit money into an account](https://trello.com/c/2MC7ExIc/28-users-can-deposit-money-into-an-account)

mileseu1 committed:
Update Customer.java

Removed ArrayList depositToAccounts, replaced with MenuPrinter.printDepositTo(accounts)
